### PR TITLE
Refactored Comments API 

### PR DESCRIPTION
**Pull request by allouis**
🍦

### CODE REVIEW

File: ghost/core/core/server/api/endpoints/comments-comments.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
         },
         permissions: true,
         query(frame) {
-            return models.Comment.findPage(frame.options);
```
Comment (ErisDS) [inline ghost/core/core/server/api/endpoints/comments-comments.js:37]: If and when you get chance, this file should prob be called members-comments.js or comments-members.js - comments-comments is left over from when we were going to have a separate comments api, but this is now officially part of the members api

Comment (allouis) [reply ghost/core/core/server/api/endpoints/comments-comments.js:37]: Awesome - I'll sort that out in this PR

Comment (allouis) [reply ghost/core/core/server/api/endpoints/comments-comments.js:37]: I'm going to go with `comments-members` as that matches the pattern we have for the content api with `resource-apitype` e.g. `posts-public`

Comment (ErisDS) [reply ghost/core/core/server/api/endpoints/comments-comments.js:37]: 👌

**Pull request merged**